revolv/templates/project/project.html

```diff
@@ -13,223 +14,344 @@
+                                                    </span>
+                                                    <span class="pull-right">
+                                                        <span class="bold">{{ project.days_left }}</span> Days Left
+                                                    </span>
+                                                </div>
+                                                <!-- end .blue-bar -->
+                                                <div class="dark-blue-bar">
+                                                    <span class="pull-left actual-energy">
+                                                        <span class="bold">{{ project.actual_energy }}</span> lbs CO
+                                                        <sub>2</sub> Avoided
+                                                    </span>
+                                                    <span class="pull-right">
+                                                        <span class="bold">{{ donor_count }}</span> {{ settings.revolv_cms.ProjectPageSettings.donors_wording }}
+                                                    </span>
+                                                </div>
+                                                <!-- end .blue-bar -->
+                                            </div>
+                                            <!-- end .info-main -->
+                                            <!-- end .video-main -->
+                                        </div>
+                                    </div>
+                                    <!-- end .info-section -->
+                                </div>
+                                <!-- end .details-active-project-module -->
+                                <!-- basic $ 20 div start here -->
+                                <div class="col-sm-12 outer_basic_silver_gold project-updates-module">
+                                    <div class="container">
+                                        <div class="col-lg-8 col-md-8 col-sm-12 col-xs-12 outer_sil_gold">
+                                            <div class="col-sm-12">
+                                                <p class="content_gold_silver">Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi.  </p>
```

Comment:
`Lorem ipsum` is dummy text. Why is it here? Please ask Andreas to provide you with the actual text if you need it.